packages/kickjs/README.md

```
@@ -1,58 +1,175 @@
+// Always export the app. The Vite plugin reads this symbol to wire HMR;
+// skipping `export` works in production but degrades dev mode to full restarts.
 export const app = await bootstrap({ modules })
 ```
 
+That's it — `http://localhost:3000/api/v1/users` returns Alice; `POST /api/v1/users` with `{ name, email }` validates against the Zod schema.
+
+## Project Layout
+
+`kick new` produces this convention; everything except the bootstrap entry is configurable via `kick.config.ts`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)